source/release-notes/changelogs/backend/changelog-backend-2022.rst

```diff
@@ -1,3 +1,13 @@
+.. _backend_20221102:
+
+2 November 2022 Release
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- :ref:`Function context.app.id <context-app>` returns a string (formerly BSON ObjectId).
+- Support :ref:`App Services deployments <deployment-regions>` in the AWS region
+  London (aws-eu-west-2).
+- Added Admin API Endpoint for App metrics.
```

Comment:
Should/can we link to this endpoint? I think it's the one added in #180

Reply:
i didn't link to it b/c it's not a ref. but let me link to ref for Admin API + specify the exact name used there.